### PR TITLE
Fix #251 Bug : Programs Page buttons not designed properly

### LINKED ISSUE #251: Bug : Programs Page buttons not designed properly
### Describe the bug
The icons and text are not aligned properly and the text is not clickable.

### To Reproduce
1. Go to 'https://anitab-org.github.io/'
2. Click on 'programs tab'
3. See error

### Expected behavior
The icon and text should be aligned. The text and icon can be clickable instead of only icon.

### Screenshots
image

Desktop (please complete the following information):

    OS: Windows 10
    Browser Firefox
    Version 85.0.1 (64-bit)

### PR DESCRIPTION
**Pull request by darshanhande11**
### Description
Made the Icon and Text clickable as a whole and changed a bit of styling to make them look clickable.

Fixes #251 

### Type of Change:

- Code
- User Interface

### Before:
before

### After:
program-page

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by darshanhande11** [comment]
@keshakaneria @codesankalp Please review my PR.

**Review by keshakaneria — CHANGES_REQUESTED** [review]
Can you align them a bit more properly to bring the arrow keys at the center of the text.
(Like "FUTURE ->", the arrow key currently is a bit above from the central horizontal axis of text, we need it to be properly aligned)

**Comment by darshanhande11** [comment]
@keshakaneria I have done the suggested change, please review it.

**Review by keshakaneria — APPROVED** [review]
Awesome 🎉Looks great!

File: src/Components/Programs/styles.js [hunk 1]
```diff
@@ -58,12 +58,22 @@ export const ArrowNavigation = styled(View,{
...
-    marginRight:8
+    marginRight:8,
+    cursor: 'pointer',
+    color: '#42aaf5',
+})
+export const ArrowLeftOutlined = styled(View,{
+    color: '#42aaf5',
+})
+export const ArrowRightOutlined = styled(View,{
+    color: '#42aaf5',
```
Comment (devkapilbansal) [inline src/Components/Programs/styles.js:76]: Can't we have only one of those as both are same
```suggestion
export const ArrowOutlined = styled(View,{
    color: '#42aaf5',
})
```

File: src/Components/Programs/Timeline.js [hunk 2]
```diff
@@ -38,14 +38,22 @@ function Timeline() {
                 <ArrowNavigation>
                     <View style={{flexDirection:'row'}}>
                         <TouchableWithoutFeedback onPress={scrollLeft}>
-                            <ArrowLeftOutlined style={{color:'#42aaf5'}}/>
+                            <View style={{flexDirection:'row'}}>
+                                <View style={{paddingTop: 3}}>
+                                    <ArrowLeftOutlined/>
+                                </View>
+                                <ArrowText>PAST</ArrowText>
+                            </View>
```
Comment (devkapilbansal) [inline src/Components/Programs/Timeline.js:46]:
```suggestion
                            <View style={{flexDirection:'row'}}>
                                <View style={{paddingTop: 3}}>
                                    <ArrowOutlined/>
                                </View>
                                <ArrowText>PAST</ArrowText>
                            </View>
```

File: src/Components/Programs/Timeline.js [hunk 3]
```diff
@@ -38,14 +38,22 @@ function Timeline() {
...
-                        <ArrowText>PAST</ArrowText>
                     </View>
                     <View style={{flexDirection:'row'}}>
-                        <ArrowText>FUTURE</ArrowText>
                         <TouchableWithoutFeedback onPress={scrollRight}>
-                            <ArrowRightOutlined style={{color:'#42aaf5'}}/>
+                            <View style={{flexDirection:'row'}}>
+                                <ArrowText>FUTURE</ArrowText>
+                                <View style={{paddingTop: 3}}>
+                                    <ArrowRightOutlined/>
```
Comment (devkapilbansal) [inline src/Components/Programs/Timeline.js:54]:
```suggestion
                                    <ArrowOutlined/>
```

**Review by devkapilbansal — COMMENTED** [review]
@darshanhande11 can you please look into this once? :point_down:

**Comment by darshanhande11** [comment]
@devkapilbansal Thank you for suggesting changes! But, `ArrowLeftOutlined` and `ArrowRightOutlined` are two different imports from `ant-design/icons`. To view the import, you can view *line number 5* of `src/Components/Programs/Timeline.js`. `ArrowLeftOutlined` is used for an icon arrow pointing left and the other one is obvious. So, in `styles.css`, I have applied styling for both of them separately. Renaming it to `ArrowOutlined` won't help because both the icons are different and imported separately.

**Comment by keshakaneria** [comment]
> @devkapilbansal Thank you for suggesting changes! But, `ArrowLeftOutlined` and `ArrowRightOutlined` are two different imports from `ant-design/icons`. To view the import, you can view _line number 5_ of `src/Components/Programs/Timeline.js`. `ArrowLeftOutlined` is used for an icon arrow pointing left and the other one is obvious. So, in `styles.css`, I have applied styling for both of them separately. Renaming it to `ArrowOutlined` won't help because both the icons are different and imported separately.

Absolutely, also @devkapilbansal keeping it different would help us in future if the theme or colors or any sort of UI/UX format is changed, it will be better if separate codesare written to change accordingly. So the current code is fine i guess. What do you say?

**Comment by devkapilbansal** [comment]
Ohh Okay, I don't know much about frontend :grimacing:

**Comment by darshanhande11** [comment]
@annabauza The [suggested changes](https://github.com/anitab-org/anitab-org.github.io/pull/265#pullrequestreview-642075463) weren't supposed to be done as I have explained it [here](https://github.com/anitab-org/anitab-org.github.io/pull/265#issuecomment-824788408).

**Pull request merged**